vendor/bat-native-ledger/src/bat/ledger/internal/database/database_contribution_queue.cc

```diff
@@ -96,6 +99,13 @@ bool DatabaseContributionQueue::MigrateToV15(
   return publishers_->Migrate(transaction, 15);
 }
 
+bool DatabaseContributionQueue::MigrateToV21(
```

Comment:
This is different table than ContributionInfo. I think this is not needed

Reply:
OK, I see. Fixed now.